core/src/dunit/scala/io/snappydata/cluster/SplitClusterDUnitTestBase.scala

```diff
@@ -150,7 +150,7 @@ trait SplitClusterDUnitTestBase extends Logging {
     doTestRowTableCreation()
   }
 
-  def _testComplexTypesForColumnTables_SNAP643(): Unit = {
+  def testComplexTypesForColumnTables_SNAP643(): Unit = {
```

Comment:
@ashetkar i will add a no-op testComplexTypesForColumnTables_SNAP643 in SplitClusterDUnitSecurityTest. Your comment got deleted by mistake.